marklogic-data-hub/src/test/resources/flow-runner-test/flows/testValuesFlow.flow.json

```diff
@@ -10,6 +10,16 @@
         "targetDatabase": "data-hub-FINAL",
         "sourceQueryIsScript": true
       }
+    },
+    "2": {
+      "name": "expectsArrayStep",
+      "stepDefinitionName": "expectArray",
+      "stepDefinitionType": "CUSTOM",
+      "options": {
```

Comment:
Should there be a "sourceQuery" option here with a source query ?

Reply:
It's added by the test - the test uses different values for sourceQuery. Which ends up verifying that sourceQuery can be specified via runtime options too.